themes/envy/envy.theme.bash

```diff
@@ -6,11 +6,24 @@ SCM_THEME_PROMPT_SUFFIX="${green}|"
 
 GIT_THEME_PROMPT_DIRTY=" ${red}✗"
 GIT_THEME_PROMPT_CLEAN=" ${bold_green}✓"
-GIT_THEME_PROMPT_PREFIX=" ${green}|"
+GIT_THEME_PROMPT_PREFIX=" ${green}| "
 GIT_THEME_PROMPT_SUFFIX="${green}|"
 
+# virtualenv prompts
+VIRTUALENV_CHAR="ⓔ "
+VIRTUALENV_THEME_PROMPT_PREFIX=""
+VIRTUALENV_THEME_PROMPT_SUFFIX=""
+
+# Display virtual environment info
+function virtualenv_prompt {
+  if [[ -n "$VIRTUAL_ENV" ]]; then
+    virtualenv=`basename "$VIRTUAL_ENV"`
+    echo -e "$VIRTUALENV_CHAR$virtualenv "
+  fi
+}
```

Comment:
The built in `virtualenv_prompt` looks like this:
```
function virtualenv_prompt {
  if [[ -n "$VIRTUAL_ENV" ]]; then
    virtualenv=`basename "$VIRTUAL_ENV"`
    echo -e "$VIRTUALENV_THEME_PROMPT_PREFIX$virtualenv$VIRTUALENV_THEME_PROMPT_SUFFIX"
  fi
}
```
You should be able to just use it and set your PREFIX and SUFFIX accordingly:
```
VIRTUALENV_THEME_PROMPT_PREFIX="ⓔ "
VIRTUALENV_THEME_PROMPT_SUFFIX=" " # trailing space
```

NOTE: I didn't try it locally but looks like it should work.

Reply:
Thanks david, I try is, it will good at this, the built in `virtualenv_prompt` works well for me, I think our user will like it too:

![Screen Shot 2020-03-19 at 9 23 09 PM](https://user-images.githubusercontent.com/1124817/77072393-93417c80-6a28-11ea-9364-2b5ceca0e81e.png)